test/helpers/expectEvent.js

```diff
@@ -27,6 +27,16 @@ async function inTransaction (tx, eventName, eventArgs = {}) {
   return inLogs(logs, eventName, eventArgs);
 }
 
+function toSimpleValue (value) {
+  return isBigNumber(value)
+    ? value.toNumber()
```

Comment:
This may cause issues for large numbers, which JavaScript won't be able to properly represent. What if instead of converting to JS number types, we instead used `should.equal` and `should.be.bignumber.equal` respectively?